docs/src/api/class-electronapplication.md

```diff
@@ -41,6 +41,26 @@ const { _electron: electron } = require('playwright');
 
 This event is issued when the application closes.
 
+## event: ElectronApplication.console
+* since: v1.42
+- argument: <[ConsoleMessage]>
+
+Emitted when JavaScript within the Electron main process calls one of console API methods, e.g. `console.log` or `console.dir`. Also emitted if the page throws an error or a warning.
```

Comment:
Why is this also emitted when the page throws an error or a warning? I guess I also missed the implementation of that.

Reply:
The test I see explicitly does `process.on('uncaughtException')`. I don't think we'll get a console message for an actual exception.

Reply:
Why is this not an actual exception?

The reason why I'm doing `process.on('uncaughtException')` is to prevent this from getting shown / blocking the main thread:

<img width="264" alt="image" src="https://github.com/microsoft/playwright/assets/17984549/3c16084a-a7a7-4c9d-85e4-d2ed871dda48">

Reply:
Removed, turns out the console.error was the reason why it was printed.